pkg/qrcodegenerator/qrcode/enddevice/lora_alliance_tr005.go

```diff
@@ -143,14 +151,52 @@ func (m *LoRaAllianceTR005) UnmarshalText(text []byte) error {
 	return m.Validate()
 }
 
-// AuthenticatedEndDeviceIdentifiers implements the AuthenticatedEndDeviceIdentifiers interface.
-func (m *LoRaAllianceTR005) AuthenticatedEndDeviceIdentifiers() (joinEUI, devEUI types.EUI64, authenticationCode string) {
-	return m.JoinEUI, m.DevEUI, m.OwnerToken
+// GetEntityOnboardingData implements the Data interface.
+func (m *LoRaAllianceTR005) GetEntityOnboardingData() *ttnpb.EntityOnboardingData {
+	paths := []string{
+		"ids",
+		"claim_authentication_code",
+	}
+	attributes := make(map[string]string)
+	if m.SerialNumber != "" {
+		attributes[serialNumberAttribute] = m.SerialNumber
+	}
+	if m.VendorID != [2]byte{} {
+		attributes[vendorIDAttribute] = strings.ToUpper(hex.EncodeToString(m.VendorID[:]))
+	}
+	if m.ModelID != [2]byte{} {
+		attributes[profileIDAttribute] = strings.ToUpper(hex.EncodeToString(m.ModelID[:]))
+	}
+	if len(attributes) > 0 {
+		paths = append(paths, "attributes")
+	}
+	return &ttnpb.EntityOnboardingData{
+		FormatId: formatIDLoRaAllianceTR005,
+		Data: &ttnpb.EntityOnboardingData_EndDeviceTempate{
```

Comment:
As far as I've seen, the `MappingKey` field of the `EndDeviceTempate` is for format conversions so I've left that out here.